src/pipeline.ml

```diff
@@ -586,4 +632,29 @@ module Mirage = struct
     Current.all @@
       ((List.map build_unikernel @@ unikernel_services ?app ())
       @ docker_services)
+
+  let deployer = {pipeline = v; admins}
 end
+
+(* cmdliner term to select the deployer to run *)
```

Comment:
The wrapping module is removed here, because we no longer need to expose the polymorphic variant values outside of this module, we just need to provide a way for the executables to be able to select out which deployer they want to run.